### DISCUSSION TITLE
any diet index approach in CHNS?

### DISCUSSION THREAD

**Discussion by ouyang363** [jamesjiadazhan/dietaryindex#52] [Q&A] [answered]
Very great package! I'm wondering is there any chance that you have any function readily available for the CHNS (China Health and Nutrition Survey) cohort?

**Comment by jamesjiadazhan** [comment, marked as answer]
Hi, thanks for reaching out. I am not familiar with the CHNS cohort, so I don't plan to design a specific function for CHNS recently. You can use the general function (e.g. HEI2015, AHEI) by providing the serving sizes by yourself from CHNS. The serving size guideline is available in the Excel sheet (see the last file on the GitHub page). You have to produce the right serving size to generate the correct results.

If you want to collaborate to provide functions for CHNS, we can talk more later if needed.

You can star the package to keep informed about the package's update.

**Comment by jamesjiadazhan** [comment]
By the way, if you wish to use the generic functions for the CHNS, you can reach out to me, so I can double-check if you use the generic functions appropriately.

**Comment by jamesjiadazhan** [comment]
Hi there, I tried to find dietary information in the CHNS, but I only found the macronutrient data, which only has carb, protein, and fat. Do you know where I can find the data from the FFQ and Nutrition survey in the CHNS?
